Packs/CommonScripts/Scripts/ExtractInbetween/ExtractInbetween.py

```diff
@@ -0,0 +1,47 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+import re
+
+
+def iter_value(value: str, character: str, incr: bool) -> list:
+    indexes = []
+    for x in range(0, len(value)):
+        if value[x] == character:
+            if incr:
+                indexes.append(x + 1)
+            else:
+                indexes.append(x)
+    return indexes
+
+
+def extract_inbetween(value: str, start: str, end: str) -> str:
+    if isinstance(value, str):
```

Comment:
Yes, it could be a JSON dict, in which case we simply return the same dictionary. I think I will change this so that if it is not a string, it will error.